server.js

```diff
@@ -123,6 +123,18 @@ module.exports.initExpress = function () {
     base: 10,
     round: 0,
   });
+  app.post(
+    '/pl/scan_paper',
+    // Exception for PDF uploads, which may be quite a bit bigger than other file uploads
+    multer({
+      storage: multer.memoryStorage(),
```

Comment:
Ah - is this just here for testing?

Reply:
Nope! It had to be at this point in the server.js file or else it wouldn't work. I am not sure what is going on under the hood.

Also, what upload size is reasonable? We could probably go to 50mb? Maybe that's too much to start with though.

Reply:
I was mostly asking about the `memoryStorage` bit - normally I'd expect to see these persisted to disk. Is this here because the raw PDF isn't actually persisted, but instead split up into pages that are stored separately? If that's the case, let's add that in a comment here.

Reply:
Yep, that's totally it. Done. 50MB + comment here.